src/OrderBook.test.tsx

```diff
@@ -1,5 +1,155 @@
-describe('<OrderBook />', () => {
-  it('should pass', () => {
-    expect(true).toBe(true);
-  });
+import React from 'react';
+import { render } from '@testing-library/react';
+import { OrderBook, Props, Layout } from './OrderBook';
+
+const book = {
+  type: 'snapshot',
+  product_id: 'BTC-USD',
+  asks: [
+    ['1166441.89', '7.3843470214'],
+    ['1162261.90', '1.50646551300'],
+    ['1166661.89', '7.38476560214'],
+    ['11224661.90', '1.506768551300'],
+  ],
+  bids: [
+    ['1165461.89', '7.384334470214'],
+    ['11676661.90', '1.5065681300'],
+    ['11643461.89', '7.38470457214'],
+    ['116564761.90', '1.50651300'],
+  ],
+};
+const propsSample: Props = {
+  applyBackgroundColor: false,
+  askColor: [235, 64, 52],
+  bidColor: [0, 216, 101],
+  book,
+  fullOpacity: false,
+  interpolateColor: () => [235, 64, 52],
+  layout: Layout.Row,
+  listLength: 2,
+  showHeaders: true,
+  showSpread: true,
+  spread: 'Hello',
+  stylePrefix: 'test_OrderBook',
+};
+
+it('OrderBook should be rendered', () => {
+  const renderedOutput = render(<OrderBook {...propsSample} />);
+
+  expect(renderedOutput).toBeTruthy();
```

Comment:
Instead of `toBeTruthy` here, can we use the matchers provided by https://github.com/testing-library/jest-dom?

It has a handy [`toBeInTheDocument`](https://github.com/testing-library/jest-dom#tobeinthedocument) matcher that is a bit more robust when checking if DOM elements did in fact get rendered. Plus it reads a bit nicer, since we are actually checking if this element got rendered into the DOM.